### ISSUE TITLE
Image needs a lot of shrinking

### ISSUE THREAD

**Issue by samip5** [linkwarden/linkwarden#353]
**Describe the bug**
I don't know if it's counted as a bug, but the image is 3-4GB which is a lot, and most of it comes from the yarn install stage.

**To Reproduce**
1. Build it
2. See that the container is HUGE.

**Expected behavior**
I would have expected the container to not be that large.

**Additional context**

<img width="1434" alt="Screenshot 2023-12-18 at 9 15 38" src="https://github.com/linkwarden/linkwarden/assets/1703002/53371eaf-5992-4cdd-ad7b-d29679e66b75">
<img width="1469" alt="Screenshot 2023-12-18 at 9 26 20" src="https://github.com/linkwarden/linkwarden/assets/1703002/cdf91c3a-2005-40e8-82d8-f7d7f887b014">

**Comment by daniel31x13** [comment]
The large container isn’t really a “bug”.

Also the CVEs are mainly related to the container itself, rather than the actual app.

**Comment by samip5** [comment]
> Also the CVEs are mainly related to the container itself, rather than the actual app.

Yes, but if one uses e.g. no-install-recommends, it shouldn't even have xorg-server package installed which currently does which the baseimage does not have thus relevant to the building of the container in this repo.
